src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1049,18 +1054,18 @@ private static BoundExpression MakeLiteral(SyntaxNode syntax, ConstantValue cons
                     var unusedDiagnostics = DiagnosticBag.GetInstance();
 
                     argument = GetDefaultParameterValue(syntax,
-                        parameterOfOptionalParametersMethod, 
-                        enableCallerInfo: ThreeState.Unknown, 
-                        localRewriter: null, 
-                        binder: binder, 
+                        parameterOfOptionalParametersMethod,
+                        enableCallerInfo: ThreeState.Unknown,
+                        localRewriter: null,
+                        binder: binder,
                         diagnostics: unusedDiagnostics);
                     kind = ArgumentKind.DefaultValue;
 
-                    unusedDiagnostics.Free();                                                                                       
-                }                                                              
+                    unusedDiagnostics.Free();
+                }
 
-                argumentsBuilder.Add(CSharpOperationFactory.CreateArgumentOperation(kind, parameter, CSharpOperationFactory.Create(argument)));
-            }                                                               
+                argumentsBuilder.Add(CSharpOperationFactory.CreateArgumentOperation(kind, parameter, operationFactory.Create(argument)));
```

Comment:
Again, probably fine for now, but it would be good to open an issue to follow up.